examples/blake_zero.rs

```diff
@@ -0,0 +1,34 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) DUSK NETWORK. All rights reserved.
+
+use blake3::{Hash, Hasher};
+
+const MAX_HEIGHT: usize = 32;
+const ARITIES: &[usize] = &[2, 4, 8];
+
+fn main() {
+    let mut hashes = Vec::with_capacity(MAX_HEIGHT + 1);
+    let zero_leaf = *b"Dusk Network -- Defendit Numerus";
+
+    for a in ARITIES {
+        let mut empty_hash = Hash::from(zero_leaf);
+        for _ in 0..MAX_HEIGHT {
+            hashes.push(empty_hash);
+            let mut hasher = Hasher::new();
+            for _ in 0..*a {
+                hasher.update(empty_hash.as_bytes());
+            }
+            empty_hash = hasher.finalize();
+        }
+        hashes.push(empty_hash);
```

Comment:
I think for the sake of an example, it makes sense when the zero-hash for a specific height is also stored in the vector at that height. You can do that by counting `height` from `0..MAX_HEIGHT` in reverse and instead of pushing the value, store it in the vector at `height`. At this point it might even makes sense to use an array instead of a vec.